src/services/klantinteracties/service.ts

```diff
@@ -243,3 +251,229 @@ export function fetchBetrokkene(url: string) {
+  const response = await fetchLoggedIn(url, {
+    method: "GET",
+    headers: {
+      Accept: "application/json",
+    },
+  });
+
+  throwIfNotOk(response); 
+  return await response.json(); 
+}
+
+function mapActorType(typeOrganisatorischeEenheid: "groep" | "afdeling" | undefined) {
+  switch (typeOrganisatorischeEenheid) {
+    case "afdeling":
+      return { codeObjecttype: "afd", codeRegister: "obj", codeSoortObjectId: "idf", soortActor: "organisatorische_eenheid" };
+    case "groep":
+      return { codeObjecttype: "grp", codeRegister: "obj", codeSoortObjectId: "idf", soortActor: "organisatorische_eenheid" };
+    default:
+      return { codeObjecttype: "mdw", codeRegister: "obj", codeSoortObjectId: "idf", soortActor: "medewerker" };
+  }
+}
+
+export async function postActoren({
+  fullName,
+  identificatie,
+  typeOrganisatorischeEenheid,
+}: {
+  fullName: string;
+  identificatie: string;
+  typeOrganisatorischeEenheid: "afdeling" | "groep" | undefined;
+}): Promise<string> {
+  const { codeObjecttype, codeRegister, codeSoortObjectId, soortActor } = mapActorType(typeOrganisatorischeEenheid);
+
+  const parsedModel = {
+    naam: fullName,
+    soortActor,
+    indicatieActief: true,
+    actoridentificator: {
+      objectId: identificatie,
+      codeObjecttype,
+      codeRegister,
+      codeSoortObjectId,
+    },
+  };
+
+  const response = await fetchLoggedIn(klantinteractiesActoren, {
+    method: "POST",
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(parsedModel),
+  });
+
+  throwIfNotOk(response);
+  const jsonResponse = await response.json();
+  return jsonResponse.uuid;
+}
+
+export const saveKlantContact = async (
+  data: KlantContactPostmodel,
+): Promise<SaveKlantContactResponseModel> => {
+  const response = await postKlantContact(data);
+  const responseBody = await response.json();
+
+  throwIfNotOk(response);
+  return { data: responseBody };
+};
+
+const postKlantContact = (data: KlantContactPostmodel): Promise<Response> => {
+  return fetchLoggedIn(`/api/postklantcontacten`, {
+    method: "POST",
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(data),
+  });
+};
+
+export const saveDigitaleAdressen = async (
+  digitaleAdressen: DigitaalAdresApiViewModel[],
+  verstrektDoorBetrokkeneUuid: string,
+  verstrektDoorPartijUuid?: string,
+): Promise<Array<{ uuid: string; url: string }>> => {
+  const savedAdressen: Array<{ uuid: string; url: string }> = [];
+
+  for (const adres of digitaleAdressen) {
+    const postBody = {
+      verstrektDoorBetrokkene: { uuid: verstrektDoorBetrokkeneUuid },
+      verstrektDoorPartij: null, 
+      adres: adres.adres,
+      soortDigitaalAdres: adres.soortDigitaalAdres === 'telefoonnummer'
+        ? 'telnr'
```

Comment:
hoe zijn deze waardes bepaald?

Reply:
Staat hier beschreven https://github.com/Klantinteractie-Servicesysteem/KISS-frontend/issues/804

Reply:
ik zie nergens 'telnr' staan in #804

Reply:
sytske is hiermee bezig om het ergens neer te zetten (duidelijk in de story)

Reply:
https://dimpact.atlassian.net/browse/PC-372 hier kan je telnr en e-mail terugvinden, dit zal sytske ook nog in de story zetten